what_the_fec/templates/candidate_office_records.j2

```diff
@@ -0,0 +1,87 @@
+{% extends 'base.j2' %}
+
+
+{% block head %}
+    <h1>{{table_name}}</h1>
+{% endblock %}
+
+
+{% block body %}
+<div id="browse" style="display: block">
+    <div class="scrollit">
+        <table>
+        <thead>
+        <tr>
+            {% for key in items[0].keys() %}
+                <th>{{ key }}</th>
+            {% endfor %}
+        </tr>
+        <tbody>
+
+        {% for item in items %}
+        <tr>
+            {% for key in item.keys() %}
+                <td>{{item[key]}}</td>
+            {% endfor %}
+
+            <td><a href="/edit_{{table_name}}/{{item.id}}" class="btn btn-default">Edit</a></td>
+            <td><a href="/delete_{{table_name}}/{{item.id}}" class="btn btn-default">Delete</a></td>
+
+        </tr>
+        {% endfor %}
+        </table>
+    </div>
+</div> <!-- browse -->
+
+<p>&nbsp;</p>
+
+{% for entries in dropdown_items_for_add.items() %}
```

Comment:
lines 38-40 can be removed (used for debugging)

Reply:
ok. please grab this one in your pr for ui updates if we should remove it.